src/gmail/gmail_endpoint.bal

```diff
@@ -724,7 +730,9 @@ public type Client client object {
 
 # Object for Spreadsheet configuration.
 #
-# + clientConfig - The http client endpoint
+# + oauthClientConfig - OAuth client configuration.
+# + httpClientConfig - HTTP client configuration.
 public type GmailConfiguration record {
-    oauth2:DirectTokenConfig clientConfig;
+    oauth2:DirectTokenConfig oauthClientConfig;
+    http:ClientConfiguration httpClientConfig = {};
```

Comment:
Do we need to expose the complete http client config to the end-user?

Reply:
Changed to obtain only the ClientSecureSocket config.